mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -303,3 +312,21 @@ def verify_flash_success(self, new_target, target, tail):
 
         self.logger.debug("ready")
         return EXIT_CODE_SUCCESS
+
+    def _check_mount_point_disappear(self, mount_point):
+        # check if mount point disappear after file copied into device
+        mount_point_previous_mounted = None
+
+        for _ in range(self.MOUNT_POINT_DISAPPEAR_TIMEOUT):
+            if os.path.ismount(mount_point):
+                mount_point_previous_mounted = True
+
+            if not os.path.ismount(mount_point) and mount_point_previous_mounted is True:
```

Comment:
could there be case where this logic doesn't work:
1) right after drag'n'drop we check `ismount`which gives `True`- but wasn't yet unmounted
2) this line doesn't match and we go to sleep(1)
.. meanwhile it unmounts and mounts back  quickly (<1s)
3) after that this line never be true because we didn't recognise ever that it was unmounted -> finally we reach timeout and raise `MountPointDisappearTimeoutError`.... hmm.. I've bad feeling about this..

Reply:
in theory can be, but from testing the disappear time take more than 1 seconds.  Still it might be, but I am thinking do we even need to check it here..

The whole flashing procedures currently are:
1. copy file
2.  _check_mount_point_disappear
3. MountVerifier(self.logger).check_points_unchanged(target)
4.  check re-mount  (FlasherMbed.runner)
5.verify_flash_success (check no Fail.txt or no Assert.txt)

so, since the re-mount will check the mount point, and  verify_flash_success  also verifies?

Reply:
if needed,  the perfect way would be:
```
while True:
   if os.path.ismount(mount_point):
           continue
   else: 
           break.
```
but what if mount point never disappear?

Reply:
check [here](https://github.com/ARMmbed/mbed-flasher/blob/e858022601a170046ce954819e789344a09b5cea/mbed_flasher/flashers/FlasherMbed.py#L117): the re-mount check sleep(2)

Reply:
To ensure that a remount has occurred you can check the **Remount count:** field in details.txt. It will increment on every remount.

Reply:
that would be good - only problem is that `remount_count` value exists only for original Daplink supported boards.